docs/roadmap-2021.md

```diff
@@ -0,0 +1,184 @@
+# Eventing
+## Next 3 Months (Short-term)
+### Event Discovery
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+**Owner:** Grant Rodgers
```

Comment:
I believe we (Google) need more time to think about our preferred approach. @markito you expressed interest in this item. Would you be able to take over ownership? If not I think it makes sense to punt it to the 6 month timeframe.

@Haolicopter @eclipselu please correct me if I'm wrong

Reply:
yes Red Hat is interested in this topic and it's taking a big portion of what we want to work on for 2021. @lberk you've this already under your radar, wdyt?